### ISSUE TITLE
Error on rename in tsx

### ISSUE THREAD

**Issue by heygwangjin** [smjonas/inc-rename.nvim#64]
Hello!

Some errors occurred when renaming the tailwindcss utility class and text in the react component. (*.tsx)

Hopefully the video explains it better. I use `noice.nvim`

https://github.com/user-attachments/assets/851af02b-1692-4a11-aa5b-1bf7ce267887

This is the neovim version I'm running
```zsh
NVIM v0.10.2
Build type: Release
LuaJIT 2.1.1727870382
```

This is my `:LspInfo`
![Screenshot 2024-10-06 at 1 24 40 AM](https://github.com/user-attachments/assets/f9fb6d87-0f01-4253-8490-636f40259ac5)

**Comment by smjonas** [comment]
Hey, can you provide me with the project files so I can try reproducing this bug? I tried to rename it with the file in the recording, but didn't get an error:

![inc-rename-tsx](https://github.com/user-attachments/assets/04bf61ba-ccc3-40a7-8321-6d502b58315a)

Also, this seems to be a Neovim bug as the error is in a Nvim internal file, can you run please run `:=vim.lsp.buf.rename("newname", {})` while your cursor is placed directly above "relative"?

If there is still an error, it means the error is not related to this plugin and you should report it to the Neovim repo directly.

**Comment by heygwangjin** [comment]
Thanks for replying @smjonas 

This is my github link of this project below. `/src/app/layout.tsx`
https://github.com/heygwangjin/v1

I ran this command `:=vim.lsp.buf.rename("newname", {})` you said, but maybe it doesn't work well.

Should I report it to the Neovim repo?

https://github.com/user-attachments/assets/60effa33-cce1-4eb9-8dfc-2efb2f771027

**Comment by smjonas** [comment]
Yes, I think it would be best to report it to the Neovim repo using a minimal configuration + language server. You should also provide the full stacktrace which you can see using the `:mes` command.

> And why Argument required error occurs when I rename normal text like Hello?

When does this occur?

**Comment by heygwangjin** [comment]
> And why Argument required error occurs when I rename normal text like Hello?

You can see the situation I mentioned at the first video I recorded. (from `0:00` ~ `0:03`)

**Comment by smjonas** [comment]
This should only happen when you submit the command `:IncRename` directly without any arguments
